src/ApiService/ApiService/Program.cs

```diff
@@ -33,8 +34,12 @@ public static void Main()
         .ConfigureServices((context, services) =>
             services
             .AddSingleton<ILogTracerFactory>(_ => new LogTracerFactory(GetLoggers()))
-            .AddScoped<ILogTracer>(s => s.GetService<LogTracerFactory>()?.MakeLogTracer(Guid.NewGuid()) ?? throw new InvalidOperationException("Unable to create a logger"))
```

Comment:
@stishkin this is where i tried to initialized the IlogTracer so that we have one instance with shared for the duration of the request so that every controller involved in the request shares the same correlationId. 
When using this the log did not appear locally when i run it.

Reply:
yup, I think I know how to fix this, I'll submit separate PR for that